### PR TITLE
Fix IP exceptions referencing the old host var

### PR DESCRIPTION
**Pull request by bdraco**
We now have multi-ip support but when I refactored, I missed some old `.host` variables in the exception text

### CODE REVIEW

**Comment by bdraco** [comment]
Issue https://github.com/home-assistant/core/issues/106531

**Review by Jc2k — APPROVED** [review]

File: aiohomekit/controller/ip/pairing.py [hunk 1]
```diff
@@ -146,18 +146,18 @@ async def _ensure_connected(self):
...
             raise AccessoryDisconnectedError(
-                f"Error while connecting to device {connection.host}:{connection.port}: "
+                f"Error while connecting to device {connection.hosts}:{connection.port}: "
                 f"{last_connector_error} ({type(last_connector_error).__name__})"
             )
 
         if not connection.is_connected:
             raise AccessoryDisconnectedError(
-                f"Ensure connection returned but still not connected: {connection.host}:{connection.port}"
+                f"Ensure connection returned but still not connected: {connection.connected_host}:{connection.port}"
```
Comment (bdraco) [inline aiohomekit/controller/ip/pairing.py:160]:
```suggestion
                f"Ensure connection returned but still not connected: {connection.hosts}:{connection.port}"
```

**Comment by Jc2k** [comment]
(Feel free to merge/release this and other fixes for beta - busy with family so won't be as reliable for acks).

**Comment by bdraco** [comment]
> (Feel free to merge/release this and other fixes for beta - busy with family so won't be as reliable for acks).

Hopefully this is the only one. 🤞

**Pull request merged**